roles/artifacts/tasks/main.yml

```diff
@@ -65,6 +65,11 @@
   ansible.builtin.import_role:
     name: os_must_gather
 
+- name: Collect crc logs when tests failed
+  ignore_errors: true  # noqa: ignore-errors
+#  when: not cifmw_success_flag.stat.exists
```

Comment:
I think this was just used for testing, now we are collecting crc logs on every run, not only failed ones which doesn't match the commit message.